### PR TITLE
Dependencies fix

### PR DESCRIPTION
**Pull request by hlogeon**
Hi! I've been playing around with your solution but clashed some problems during installation. Maybe I'm doing something wrong but when I try to follow your installation instructions, `composer install` fails with errors like:
```
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - The requested package phpspec/prophecy dev-patch-1 as 1.6.2 exists as phpspec/prophecy[v1.1.2, v1.4.0, dev-master, 1.7.x-dev, v1.0.0, v1.0.0-BETA1, v1.0.0-BETA2, v1.0.1, v1.0.2, v1.0.3, v1.0.4, v1.1.0, v1.1.1, v1.2.0, v1.2.1, v1.3.0, v1.3.1, v1.4.1, v1.5.0, v1.6.0, v1.6.1, v1.6.2, v1.7.0, 1.6.x-dev] but these are rejected by your constraint.
```

And there are some other issues in composer.json files which leads to errors. I just replaced previously specified versions by the newer and MORE STABLE versions of the packages. I hope it can help to someone like me who wants just to play with your solution without any modifications.

### CODE REVIEW

File: service/user-read/composer.json [hunk 1]
```diff
@@ -21,13 +21,13 @@
   ],
   "require": {
     "php": "^7.1",
-    "prooph/common": "4.0.x-dev",
-    "prooph/event-store": "7.0.x-dev",
-    "prooph/pdo-event-store": "1.0.x-dev"
+    "prooph/common": "^4.0",
+    "prooph/event-store": "^7.0",
```
Comment (prolic) [inline service/user-read/composer.json:25]: Revert this please

File: service/user-read/composer.json [hunk 2]
```diff
@@ -21,13 +21,13 @@
...
+    "prooph/pdo-event-store": "^1.0"
```
Comment (prolic) [inline service/user-read/composer.json:26]: Revert this please

File: service/user-read/composer.json [hunk 3]
```diff
@@ -21,13 +21,13 @@
...
   },
   "require-dev": {
     "phpunit/phpunit": "^5.7",
-    "phpspec/prophecy": "dev-patch-1 as 1.6.2",
+    "phpspec/prophecy": "1.7.0",
```
Comment (prolic) [inline service/user-read/composer.json:30]: Should be ^1.7

**Review by prolic — CHANGES_REQUESTED** [review]

File: service/user-read/composer.json (hunk 1 above)
Comment (hlogeon) [reply service/user-read/composer.json:25]: Did you check it? If I revert it it's just stop working, I think there is not such tags in your repos anymore.

File: service/user-read/composer.json (hunk 2 above)
Comment (hlogeon) [reply service/user-read/composer.json:26]: Did you check it? If I revert it it's just stop working, I think there is not such tags in your repos anymore.

File: service/user-read/composer.json (hunk 3 above)
Comment (hlogeon) [reply service/user-read/composer.json:30]: True, got it

File: service/user-read/composer.json (hunk 1 above)
Comment (prolic) [reply service/user-read/composer.json:25]: There is no 7.0 final release yet. Use 7.0-dev, if that breaks it, we need to update micro-do.

**Pull request closed without merging**